### PR TITLE
KSF-88: Getting started - fixed upon developer review.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

**Review by Slash32 — CHANGES_REQUESTED** [review]
I would not recommend dropping whole the TIP part with description of the Network configuration, modes, etc.

Could this be just extended with the information that the Sandbox is now capable of working out of the box (provided the Hostname is correct)?

The main advantage of Bridged is that the Sandbox can detect and apply the Kaa Host correctly. While NAT does not provide such support and is better in other aspects.

**Comment by hunch7** [comment]
Fixed.

**Review by Slash32 — APPROVED** [review]

File: doc/Getting-started/index.md [hunk 1]
```diff
@@ -53,24 +53,17 @@ The current version of Kaa Sandbox supports Oracle VirtualBox version 5.1.2 and
...
 For optimal performance, we recommend at least 4 GB of RAM and at least 2 CPUs.
+Apply your settings and start the virtual machine.
 
-	>**TIP:** Configure your network settings.  
-	>The default VM networking configuration is NAT, which enables your host machine to connect to the [Kaa server]({{root_url}}Glossary/#kaa-server).
-	>This means that you will need to deploy your [Kaa client]({{root_url}}Glossary/#kaa-client) applications to your host machine to enable their communication with the Sandbox server.
-	>
-	>You can deploy the Kaa client applications to any hardware other than your host machine, and access Kaa web interfaces from your network devices.
+	>**NOTE:** Your instance of Kaa Sandbox works out of the box with the default VM networking configuration set to NAT.
+	>This enables your host machine to connect to the [Kaa server]({{root_url}}Glossary/#kaa-server).
```
Comment (rasendubi) [inline doc/Getting-started/index.md]: As far as I know, in current configuration this allows connecting from any machine, not only host.

File: doc/Getting-started/index.md [hunk 2]
```diff
@@ -53,24 +53,17 @@ The current version of Kaa Sandbox supports Oracle VirtualBox version 5.1.2 and
...
+	>However, you may want to switch to the **Bridged Adapter** mode.
 	>To do that, open the **Oracle VM VirtualBox Manager**, click **Settings** > **Network** and change the **Attached to** option to **Bridged Adapter**.
-	>
-	>To configure the networking mode, switch to the **Network** tab in the VM settings.
-	>Click **Advanced** to unfold the drop-down menu, and make sure the **Promiscuous Mode** option is set to **Allow All**.
-	{: .tip}
+	>The advantage of using this mode is that your Kaa Sandbox will work correctly even if you change your local machine IP.
+	{:.note}
+5. Wait until the VirtualBox starts up and open [127.0.0.1:9080/sandbox](http://127.0.0.1:9080/sandbox) URL in your browser.
```
Comment (rasendubi) [inline doc/Getting-started/index.md:64]: Does this work for bridged adapter?

Comment (hunch7) [reply doc/Getting-started/index.md:64]: No, it doesn't. Actually, the Sandbox console displays a pretty clear message regarding what IP your Sandbox is on, for both NAT and Bridged Adapter, so our dear users won't get lost anyway.

File: doc/Getting-started/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Getting-started/index.md]: Fixed.

File: doc/Getting-started/index.md (hunk 2 above)
Comment (Slash32) [reply doc/Getting-started/index.md:64]: Actually, Sandbox will work correctly provided it is in Bridged mode and gets rebooted after the IP change. Applications will not work until their SDKs are updated with the new IP address.

What is meant here is that for NAT mode, you will need to also change the Management->Kaa Host manually before downloading SDKs.

Comment (Slash32) [reply doc/Getting-started/index.md:64]: Thus, basically, the text presented is correct. It is just not full. But better explanation might be too complex.

**Review by rasendubi — APPROVED** [review]

**Pull request merged**